projects/github-actions/required-review/src/reviewers.js

```diff
@@ -22,11 +22,11 @@ async function fetchReviewers() {
 			per_page: 100,
 		} ) ) {
 			res.data.forEach( review => {
-				// Looks like GitHub may return more than one review per user, but only counts the last for each.
-				// "APPROVED" allows merging, while anything else (e.g. "CHANGED_REQUESTED" or "DISMISSED") doesn't.
+				// GitHub may return more than one review per user, but only counts the last non-comment one for each.
+				// "APPROVED" allows merging, while "CHANGES_REQUESTED" and "DISMISSED" do not.
 				if ( review.state === 'APPROVED' ) {
 					reviewers.add( review.user.login );
-				} else {
+				} else if ( review.state === 'CHANGES_REQUESTED' || review.state === 'DISMISSED' ) {
```

Comment:
I'd feel more comfortable here if we were to check for
```suggestion
				} else if ( review.state !== 'COMMENT' ) {
```
in case GitHub adds more review states in the future.

Reply:
Makes sense!
https://github.com/Automattic/jetpack/pull/26249/commits/75775358bebf5aa720fce7c2387036ec3cd1bf52